### PR TITLE
Add ethan605 keymap for kbd75

### PR DESCRIPTION
**Pull request by ethan605**
## Description

Modified the default kbd75 keymap for more personal use.

## Types of Changes

- [ ] Core
- [ ] Bugfix
- [ ] New feature
- [ ] Enhancement/optimization
- [ ] Keyboard (addition or update)
- [x] Keymap/layout/userspace (addition or update)
- [ ] Documentation

## Issues Fixed or Closed by This PR

* 

## Checklist

- [x] My code follows the code style of this project: [**C**](https://docs.qmk.fm/#/coding_conventions_c), [**Python**](https://docs.qmk.fm/#/coding_conventions_python)
- [x] I have read the [**PR Checklist** document](https://docs.qmk.fm/#/pr_checklist) and have made the appropriate changes.
- [ ] My change requires a change to the documentation.
- [ ] I have updated the documentation accordingly.
- [x] I have read the [**CONTRIBUTING** document](https://docs.qmk.fm/#/contributing).
- [ ] I have added tests to cover my changes.
- [x] I have tested the changes and verified that they work and don't break anything (as well as I can manage).

### CODE REVIEW

File: keyboards/kbdfans/kbd75/keymaps/ethan605/keymap.c [hunk 1]
```diff
@@ -0,0 +1,62 @@
+#include QMK_KEYBOARD_H
+
+#define _______ KC_TRNS
```
Comment (drashna) [inline keyboards/kbdfans/kbd75/keymaps/ethan605/keymap.c:3]: These defines aren't necessary, as these are actually defined in the core.  You can safely remove them. 

https://github.com/qmk/qmk_firmware/blob/a0309db983150fd0197eb620a6ba552d90d29f93/quantum/quantum_keycodes.h#L30-L31


```suggestion
```

File: keyboards/kbdfans/kbd75/keymaps/ethan605/keymap.c [hunk 2]
```diff
@@ -0,0 +1,62 @@
...
+  ),
+};
+
+void led_set_user(uint8_t usb_led) {
+    if (usb_led & (1 << USB_LED_CAPS_LOCK)) {
+        DDRB |= (1 << 2); PORTB &= ~(1 << 2);
+    } else {
+        DDRB &= ~(1 << 2); PORTB &= ~(1 << 2);
+    }
+}
```
Comment (drashna) [inline keyboards/kbdfans/kbd75/keymaps/ethan605/keymap.c:61]: This function is deprecated.  Could you update to use this: 
https://docs.qmk.fm/#/custom_quantum_functions?id=example-led_update_user-implementation

Also for the pin controls, we recommend/encourage using the GPIO functions: 
https://docs.qmk.fm/#/internals_gpio_control

**Review by drashna — CHANGES_REQUESTED** [review]
And could you add license headers to the `c` and `h` files in this PR?

File: keyboards/kbdfans/kbd75/keymaps/ethan605/keymap.c (hunk 2 above)
Comment (ethan605) [reply keyboards/kbdfans/kbd75/keymaps/ethan605/keymap.c]: Thanks @drashna, it seems that I don't need it anymore.

**Review by drashna — APPROVED** [review]

**Review by fauxpark — APPROVED** [review]

**Review by noroadsleft — APPROVED** [review]

**Comment by noroadsleft** [comment]
Thanks!

**Pull request merged**